### DISCUSSION TITLE
Renovate doesn't see all the packages in onboarding PR

### DISCUSSION THREAD

**Discussion by just-jeb** [renovatebot/renovate#13627] [Request Help] [answered]
Hello folks, hope you're doing well!  
I'm currently onboarding Renovate into [my repo](https://github.com/just-jeb/angular-builders/pull/1085), however according to PR description it only picks part of the package files.  
As you can [see](https://github.com/just-jeb/angular-builders/blob/master/package.json#L10) there are also `example` packages, and there is also a `yarn.lock`.  
I'm pretty sure I'm just missing something in the config, but it's huge and I have no idea what I'm looking for 😅 
I'd appreciate any help, thank you!

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

those are probably ignored by default `config:base` preset 😉

**Reply by just-jeb** [reply]
Wow, thanks for such a swift response! This should solve the issue, appreciate a lot👍
Any idea why `yarn.lock` is not in the picture? Isn't it considered a package file?  
I thought that maybe `"updateLockFiles": false` hides behind some config, but explicitly setting it to `true` didn't change anything.
